packages/venia-ui/lib/queries/addConfigurableProductsToCart.graphql

```diff
@@ -0,0 +1,29 @@
+mutation addConfigurableProductToCart(
+    $cartId: String!
+    $quantity: Float!
+    $sku: String!
+    $parentSku: String!
+) {
+    addConfigurableProductsToCart(
+        input: {
+            cart_id: $cartId
+            cart_items: [
+                {
+                    data: { quantity: $quantity, sku: $sku }
+                    parent_sku: $parentSku
+                }
+            ]
+        }
+    ) {
+        cart {
```

Comment:
Same comment as other PR; fetching this data has a cost on the backend (though I hope this primes the cache, and the next getCartDetails call serves from the cache 🤞)

Reply:
I replied in the other thread but basically there is no such thing as a `null` response in GQL. You can choose to return a success indicator boolean or some other "output" type, as we do in Magento GQL. You can check the schema, or you can try in graphiql yourself, but it will complain if you remove the output.

Reply:
And this *may* prevent the extra "get details" request from being necessary, so not completely a bad thing.